### PR TITLE
NCNotificationLongLookView Support

### LINKED ISSUE #6: Applying notification hooks to NCNotificationLongLookView
To show the progress view when Haptic/3D Touch-ing on the notification.

Note: this is more than duplicating the code and hooking NCNotificationLongLookView instead - doing so will result in having 2 different UIProgressView instances, that aren't in sync with each other and is basically inefficient. This requires using the same view, which means we might have to re-implement adding the UIProgressView to the appropriate view.

### PR DESCRIPTION
**Pull request by RuntimeOverflow**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,8 +3,7 @@ include $(THEOS)/makefiles/common.mk
 
 export FINALPACKAGE = 1
 DEBUG = 0
-export ARCHS = arm64 #arm64e
-export TARGET = iphone:clang:13.5
```
Comment (lgariv) [inline Makefile]: Necessary to compile with the right SDK using GitHub workflow. You can download mine from [lgariv/theos](https://www.github.com/lgariv/theos)

**Review by lgariv — CHANGES_REQUESTED** [review]
[reference](https://github.com/lgariv/DownloadBar14/pull/9/files#r551972029)

**Comment by RuntimeOverflow** [comment]
Ok didn't know that. For some reason it wouldn't compile if a had the target variable set (Complaining about an empty SYSROOT).

**Comment by lgariv** [comment]
> Ok didn't know that. For some reason it wouldn't compile if a had the target variable set (Complaining about an empty SYSROOT).

SYSROOT is the path of the SDK, you can set it (`export SYSROOT=/path/to/iPhoneOS.sdk`) in your `~/.bashrc`

**Review by lgariv — APPROVED** [review]

**Comment by RuntimeOverflow** [comment]
I know that's what I did, but because I thought the target wasn't necessary I removed it.

**Pull request merged**